roofit/roofitcore/inc/RooAbsData.h

```diff
@@ -51,7 +54,7 @@ class RooAbsData : public TNamed, public RooPrintable {
 
   // Constructors, factory methods etc.
   RooAbsData() ; 
-  RooAbsData(const char *name, const char *title, const RooArgSet& vars, RooAbsDataStore* store=0) ;
+  RooAbsData(std::string_view name, std::string_view title, const RooArgSet& vars, RooAbsDataStore* store=0) ;
```

Comment:
```suggestion
  RooAbsData(std::string_view name, std::string_view title, const RooArgSet& vars, RooAbsDataStore* store=0) ;
  RooAbsData(ROOT::Internal::TStringView name, ROOT::Internal::TStringView title, const RooArgSet& vars, RooAbsDataStore* store=0) : RooAbsData(std::string_view(name), std::string_view(title), vars, store) {};
```

Reply:
Hi Philippe, thanks for the suggestion! I have implemented a templated C++-14-build compatible constructor that I put in every RooFit data class with a preprocessor macro. Is this okay now?

Reply:
Nice :)